src/main/java/mate/academy/dao/ConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+import mate.academy.exeption.DataProcessingException;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/my_db";
```

Comment:
for future - bad (too abstract) db name
```suggestion
    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc-intro";
```